### PR TITLE
Adds the bababooey sound effect to the H.O.N.K soundboard

### PR DESCRIPTION
**Pull request by Toastgoats**
## About The Pull Request

Exactly what it says on the tin, it just adds a new sound to the list of sound effects the honker can make.

## Why It's Good For The Game

It's funny, just a quick little nod to my favorite joke merge without adding anything too distruptive.

## Changelog

:cl:
add: The H.O.N.K can now bababooey.
soundadd: The return of the king.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
licensing of sound?

**Comment by Toastgoats** [comment]
> licensing of sound?

Ah, guess that's what the attributions file is for, I'll get on that.

File: sound/attributions.txt [hunk 1]
```diff
@@ -28,4 +28,4 @@ splatter.ogg adapted from https://freesound.org/people/Rocktopus/sounds/233418/
 hohoho.ogg and hehe.ogg are cut from a recording by Nanakisan on freesound: https://freesound.org/people/Nanakisan/sounds/253534/
 mbox_full.ogg and mbox_end.ogg make use of The Ragtime Drummer by James Lent, in the public domain
 
-growl1.ogg and growl2.ogg in /sound/creatures/dog are adapted from Glitchedtones's Freesound shih-tzu uploads https://freesound.org/people/Glitchedtones/
+"Bababooey" sound effect is cut from and credited to The Howard Stern Show.
```
Comment (LemonInTheDark) [inline sound/attributions.txt:31]: A: Please do not accidentally delete old stuff credits  
B: Can we uh, legally use this audio?

Comment (Toastgoats) [reply sound/attributions.txt:31]: What the fuck I don't remember doing that, hold on.

**Comment by LemonInTheDark** [comment]
Have you tested this?

**Comment by Toastgoats** [comment]
> Have you tested this?

In my copy of the codebase, yes, but I guess it wouldn't hurt to try it with a more recent version.

**Comment by optimumtact** [comment]
attribution !== license to use you donut

**Comment by Toastgoats** [comment]
> attribution !== license to use you donut

Well shit, I assumed this wouldn't be much of an issue since this used to be in the codebase.
I'll leave this up until one of us definitively knows if we can use this or not.

**Comment by MrMelbert** [comment]
the HONK soundboard is being (temporarily :tm:) removed in another pr ( #65375 ) so this might not last very long

**Comment by optimumtact** [comment]
it is very unlikely we can legally use it

**Comment by Toastgoats** [comment]
> it is very unlikely we can legally use it

Sad but understandable.

**Pull request closed without merging**